core/ipc/rpc/kafka/src/main/java/org/opennms/core/ipc/rpc/kafka/KafkaRpcServerManager.java

```diff
@@ -162,6 +171,11 @@ public void init() throws IOException {
         });
         tracerRegistry.init(minionIdentity.getLocation() + "@" + minionIdentity.getId());
 
+        BulkheadConfig bulkheadConfig = BulkheadConfig.custom()
+                .maxConcurrentCalls(maxConcurrentCalls)
+                .maxWaitDuration(java.time.Duration.ofMillis(maxWaitDuration))
+                .build();
+        bulkhead = Bulkhead.of("ipc-rpc-kafka", bulkheadConfig);
```

Comment:
Let's expose the bulk head metrics like we've done here: https://github.com/OpenNMS/opennms/blob/develop/features/dnsresolver/netty/src/main/java/org/opennms/netmgt/dnsresolver/netty/NettyDnsResolver.java#L126